extensions/standard-processors/tests/unit/resources/ca_B.crt

```diff
@@ -0,0 +1,21 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
From the unit test, it looks like this is an invalid certificate.  Is that true, and if it is, then can you put "invalid" in the file name, please?

Reply:
Its not invalid per se, just the server is configured to use ca_A, so a client configured with a different CA will fail the handshake.

Reply:
I see, thanks